codebuild/spec/buildspec_generalbatch.yml

```diff
@@ -288,7 +288,7 @@ batch:
     - buildspec: codebuild/spec/buildspec_ubuntu.yml
       env:
         compute-type: BUILD_GENERAL1_LARGE
-        image: 024603541914.dkr.ecr.us-west-2.amazonaws.com/docker:ubuntu18codebuild
+        image: 024603541914.dkr.ecr.us-west-2.amazonaws.com/docker:ubuntu22codebuild
```

Comment:
Is this part of the fix?

Reply:
it updates cmake from 3.10 to 3.22, which is another factor in my inability to repro this locally.

Reply:
Will this still pass ci if we dont do this upgrade? We should upgrade the image but would be good to isolate the only fix first.

Reply:
Appears to work on the older [Cmake 3.10](https://us-west-2.console.aws.amazon.com/codesuite/codebuild/024603541914/projects/s2nGeneralBatch/build/s2nGeneralBatch%3A2c38c0f0-38eb-4722-8d35-7e5b54f9e55b/log?region=us-west-2)

Reply:
reverted.